### DISCUSSION TITLE
Dynamic Links / Deferred Deep Linking

### DISCUSSION THREAD

**Discussion by jtkeyva** [supabase/supabase#10398] [Feature Requests]
Does Supabase plan to support Dynamic Links like Firebase or Branch.io?

This is important because the same link should go to a webpage if the app is not installed or to a deep link within the app if it is installed.

It's just these "little" things that I miss about Firebase.

**Comment by blakelee** [comment]
Resurrecting this from the dead. This would be a great feature to have so that I don't need to use appsflyer or branch.io
